src/librustc_yk_sections/Cargo.toml

```diff
@@ -12,5 +12,8 @@ test = false
 [dependencies]
 rustc = { path = "../librustc" }
 rustc_yk_link = { path = "../librustc_yk_link" }
-ykpack = { git = "https://github.com/softdevteam/ykpack" }
+#ykpack = { git = "https://github.com/softdevteam/ykpack" }
```

Comment:
> There is a local ykpack Cargo path until we decide what to do with
PHIs arity

So ignore this for now. I'll have a PR for ykpack once we've talked about PHI arity.